spec/fixtures/modules/facts/plans/emit.pp

```diff
@@ -0,0 +1,4 @@
+plan facts::emit(String $host) {
+  $target = get_targets($host)[0]
+  run_command("echo 'Facts for ${host}: ${facts($target)}'", $host)
```

Comment:
Could we return the string directly (without calling the `run_command`) like this:
```
"Facts for ${host}: ${facts($target)}"
```
?

Reply:
Yeah, we could.

Reply:
I'll make all the facts and vars tests independent of transport and return values rather than echo them.

Reply:
Updated.